src/main.nf

```diff
@@ -49,15 +48,70 @@ else {
 }
 
 
+if ( params.run_manta ) {
+    process index_bamfile {
+        input:
+            file 'bamfile' from bamfile
+        output:
+            file 'bamfile.bai' into bamfile_index
+
+        module 'bioinfo-tools'
+        module "$params.modules.samtools"
+
+        """
+        samtools index bamfile
+        """
+    }
+
+    process run_manta {
+        input:
+            file 'bamfile_tmp' from bamfile
+            file 'bamfile.bai' from bamfile_index
+        output:
+            file 'sample.manta.sv.vcf.gz' into manta_vcf_gz
```

Comment:
Interesting use of the process name in the variable name :) ... actually kind of makes sense. It kind of works around the problem of these being "free-flying global variables", which you don't know where they are coming from. Is this something we should make into a recommendation maybe, for nextflow workflows?